src/vsr/replica_test.zig

```diff
@@ -1536,6 +1578,11 @@ const TestContext = struct {
         };
     }
 
+    pub fn client_bus(t: *TestContext, client_index: usize) !*TestClientBus {
+        // Reuse one of `Cluster.clients`' ids since the Network preallocated links for it.
+        return TestClientBus.init(t, t.cluster.clients[client_index].id);
```

Comment:
:exploding_head: glorious! Really like how we just put a separate bus alongside what we already have!